src/Input/Silk.NET.Input.Sdl/SdlMotor.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Silk.NET.Input.Sdl
+{
+    internal class SdlMotor : IMotor
+    {
+        private readonly SdlGamepad _gp;
+
+        public SdlMotor(int i, SdlGamepad gp)
+        {
+            _gp = gp;
+            Index = i;
+        }
+
+        public int Index { get; }
+
+        public float Speed
+        {
+            get => SpeedVal;
+            set
+            {
+                _gp.VibrationChanged = true;
+                SpeedVal = value;
+            }
+        }
+
+        public float SpeedVal { get; private set; }
```

Comment:
What's going on here?

Reply:
SpeedVal is a backing property for the Speed property. I have converted it into a backing field only as there's no reason why it needs to be public or an autoproperty.